src/renderer/widgets/WalletDetails/model/wallet-provider-model.ts

```diff
@@ -17,18 +18,83 @@ const $accounts = combine(
   },
 );
 
-const $connected = combine($accounts, walletConnectModel.$client, (accounts, client): boolean => {
-  const account = accounts[0];
-  if (!client || !account || !accountUtils.isWalletConnectAccount(account)) return false;
+const $singleShardAccount = combine(
+  {
+    accounts: walletModel.$accounts,
+  },
```

Comment:
Is object required here? You can just provide `walletModel.$accounts` and use it as single argument in callback